pkg/preflight/preflight.go

```diff
@@ -194,7 +208,7 @@ func Run(appID string, appSlug string, sequence int64, isAirgap bool, archiveDir
 			// preflight reporting
 			if isDeployed {
 				if err := reporting.WaitAndReportPreflightChecks(appID, sequence, false, false); err != nil {
-					logger.Debugf("failed to send preflights data to replicated app: %v", err)
+					logger.Errorf("failed to send preflights data to replicated app: %v", err)
```

Comment:
i'm not sure this message is always accurate anymore given the work to support airgap reporting.  since we're exposing it as a `error` log with this change, how about we change the message to something more generic like:
```
failed to report preflight check results
```

Reply:
> i'm not sure this message is always accurate anymore given the work to support airgap reporting. since we're exposing it as a `error` log with this change, how about we change the message to something more generic like:
> 
> ```
> failed to report preflight check results
> ```

I've seen a number of these debug logs all over the code. I think I'll revert this change for now and leave it as is. Later on we can align the logs.